_home-desktop-version.scss

```diff
@@ -0,0 +1,132 @@
+@media (min-width: 762px) {
+  .container {
+    max-width: 1000px;
+    margin: 0 auto;
+  }
+
+  .hamburger,
+  .mobile-menu {
+    display: none;
+  }
```

Comment:
- [ ] The contact menu on the mobile view doesn't  direct you anywhere, please make a connection with other links.
- [ ] I recommend you to use other like black instead of using red or brown color as a background color for the mobile menu to have the corresponding colors in our project.
![navbar1](https://user-images.githubusercontent.com/55635977/128437874-a2ef1c83-ad54-4e8f-a3a2-12c0fabc11f2.png)

Reply:
Well addressed this. good job!